### ISSUE TITLE
private message echoing back to sender

### ISSUE THREAD

**Issue by Charliebaby019** [YassienW/kik-node-api#9]
(hello)

its minor but here's a contribution to help me feel productive.

if im not mistaken exampleBot.js is sending back a copy of what the remote user PMs to user .
(just delete since a console.log exists at the end?)

line 57  

Kik.sendPrivateMessage(sender.jid, msg, (delivered, read) =>

**Comment by YassienW** [comment]
Well, exampleBot is just meant to contain code examples to demonstrate API usage. It isn't meant to be used for anything meaningful anyway. The bot does other absurd things such as kick anyone who joins a group and ban anyone who leaves.

**Comment by Charliebaby019** [comment]
sorry, I also realized i wasnt running the entire updated program, so i updated everything.
really just trying to keep some activity in your github so that you don't feel your efforts to supply this api are all for nothing. i've appreciated all your efforts.

and yeah, i noticed the auto kick and auto ban, commented those out.

 but 
this morning i've noticed this....

im getting this data notification every 30 minutes if no data is recieved then if data is receieved its delayed 40 minutes and then continues every 30 minutes again.... until windows reports that its run out of memory
and the logs show --- RAW: Received data (1):  
--- after the unhandled notification

also I have tried being added to a group. then after authentication , and Getting roster , it displays this unhandled data notification , and just fails to retrieve any roster information afterwords

I know i probably very often , if not everytime come off as a complete idiot in many of my post with you, i just hope you don't grow overly annoyed by my ignorance. lol. what can i say , but thanks for doing this. all of this.

![qqqq](https://user-images.githubusercontent.com/54005644/86286486-2489c880-bbac-11ea-9978-9d8c193c9637.JPG)
